pkgs/development/compilers/llvm/17/default.nix

```diff
@@ -408,6 +408,11 @@ in let
           url = "https://github.com/macports/macports-ports/raw/acd8acb171f1658596ed1cf25da48d5b932e2d19/lang/llvm-17/files/0042-mbstate_t-not-defined.patch";
           hash = "sha256-jo+DYA6zuSv9OH3A0bYwY5TlkWprup4OKQ7rfK1WHBI=";
         })
+      ] ++ lib.optionals stdenv.hostPlatform.isOpenBSD [
```

Comment:
Why conditional?

Reply:
Just to avoid rebuilds. Like before I hope we can avoid that problem by switching to 18.